src/android/UsabillaCordova.java

```diff
@@ -127,4 +135,24 @@ private void sendEvent(String eventName) {
         }
         return customVars;
     }
+
+    private void dismiss() {
+        if (Usabilla.INSTANCE.dismiss(cordova.getActivity())) {
+            callbackContext.success();
+            return;
+        }
+        callbackContext.error("No forms to dismiss");
+    }
+
+    private void setDataMasking(JSONObject data) throws JSONException {
+        final JSONArray masks = data.getJSONArray("MASKS");
```

Comment:
To be consistent with the way we have done it for other variables these could probably be specified at the top of the file in such a way

```
    private static final String MASKS = "MASKS";
    private static final String MASK_CHAR = "MASK_CHAR";
```